### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MigVega**
Suggestions to improve readme title, terminal commands and citation format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# README - Georgia Tech Smoothing and Mapping Library
+# Georgia Tech Smoothing and Mapping Library
```
Comment (varunagrawal) [inline README.md:1]: Can you please leave this as saying `README - Georgia Tech Smoothing and Mapping Library`?

**Review by varunagrawal — COMMENTED** [review]
Looks good except for 1 comment.

File: README.md (hunk 1 above)
Comment (MigVega) [reply README.md:1]: Ok!

**Review by MigVega — COMMENTED** [review]
I left: 2README - Georgia Tech Smoothing and Mapping Library", as you wanted! :)

**Pull request merged**